phpBB/language/en/acp/search.php

```diff
@@ -125,6 +125,7 @@
 	),
 	'SEARCH_INDEX_CREATED'					=> 'Successfully indexed all posts in the board database.',
 	'SEARCH_INDEX_REMOVED'					=> 'Successfully deleted the search index for this backend.',
+	'SEARCH_INDEX_PROGRESS'					=> '<br><progress value="%1$d" max="%3$d"></progress><br> %4$.2f %% <br>Done: %1$d | Pending: %2$d | Total: %3$d',
```

Comment:
It'd be better to have all HTML markup in template rather than on PHP.

Reply:
can not do that in `trigger_error(...)` function.

Reply:
do you have a way to do that ???

Reply:
Well, in theory that cab be done via adding relevant code to `\adm\style\message_body.html` and assigning related template vars before `trigger_error()` call.

Reply:
since it's a small thing,
and also the `%1$d` & `%3$d` are repetitive,
hence I added it in the lang file.